x11-misc/mugshot/mugshot-0.4.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eapi7-ver gnome2-utils
+
+MY_PV="$(ver_cut 1-2)"
+
+DESCRIPTION="A lightweight user-configuration application"
+HOMEPAGE="https://launchpad.net/mugshot"
+SRC_URI="https://launchpad.net/${PN}/${MY_PV}/${PV}/+download/${PN}-${PV}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
Did you verify it?

Reply:
Fixed.. Why nearly all sites write "GPL-X" but not "GPL-X+" -.-

Reply:
If by 'sites' you mean GitHub, then GitHub just verifies whatever file is placed as 'COPYING' or 'LICENSE'. It doesn't check exact copyright notices in source files.